src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Repository/ReleaseVersionRepositoryTests.cs

```diff
@@ -469,11 +471,11 @@ public async Task Success()
 
                 var result = await repository.ListLatestReleaseVersionIds(publication.Id, publishedOnly: PublishedOnly);
```

Comment:
I wonder if it would be better to create a private method `ListLatestReleaseVersionIds`, that feeds a value of `false` for the `publishedOnly` parameter into the method call, rather than storing the `publishedOnly` parameter in a class constant.

so you call it like `var result = await ListLatestReleaseVersionIds(repository, publication.Id)`, and you can forget about the `publishedOnly` flag.

Similar for the other test classes